### PR TITLE
add a page on decommissioning production systems

### PR DESCRIPTION
**Pull request by afeld**
Anything I'm missing?

### CODE REVIEW

**Comment by erik-burgess** [comment]
A few additional items for consideration: 

1) Is there an associated New Relic account(s)?
2) Are there any DNS changes that need to happen? 
3) Is there a custom domain and what should be done with it? 
4) Is there an SSP and should we notify GSA that the system no longer exists? 
5) There is mention of deleting AWS resources, but does the project have it's own AWS Account that should be decommissioned (this is a slightly more in depth process and should be noted in #infrastructure or via a GH issue)?

**Comment by alain-hoang** [comment]
This might be covered above but does the system have integrations with systems like slack that need to be cleaned up?

**Comment by afeld** [comment]
Wow, lots I didn't think about!

@gbinal Anything folks need to do on the api.data.gov side? @tdlowden How about for DAP?

**Comment by tdlowden** [comment]
@afeld no extra steps for DAP. Assuming the site is decommissioned, data will just stop flowing into the account.

File: _includes/decommissioning_checklist.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+* [ ] Export any production data and put it into Google Drive, for [records retention](https://handbook.18f.gov/records-management/) purposes.
+* [ ] Ask in [#cg-support](https://gsa-tts.slack.com/messages/cg-support) for the cloud.gov organization to be deleted.
```
Comment (brittag) [inline _includes/decommissioning_checklist.md]: Prefer cloud-gov-support@gsa.gov for org deletion requests (goes into ticketing system; less likely to lose track of the task).

**Comment by yozlet** [comment]
Here's the long decom checklist for MyUSA. It's worth looking through the comments too - not everything that was discussed made it onto the checklist (because it wasn't relevant to MyUSA) but it may be for other projects.

**Comment by afeld** [comment]
@yozlet See my last commit 😉

**Comment by yozlet** [comment]
Regarding the first step (notifying users): There's a wider topic here about _no longer being a dependency_ and removing the project in as thoughtful and non-damaging a way as possible. This involves understanding who's using it, why they're using it, and how much trouble the removal is going to cause. (I would say that this goes double for APIs, but we've also had web apps - such as - FBopen - that we didn't realise other gov't workers depended on.) In theory, this should all happen before the checklist is needed; in practice, you can bet that at least some new dependents will reveal themselves when you're halfway through.

I would add steps like...
 * As early as you can, all public presentation of the project should note that it is deprecated and heading for decommission. Front page banners, new signup emails (assuming signups are still allowed - why?), all repo READMEs.
 *  Create as large a list of potentially-interested parties as possible. Got a `users` table? How about API key grants? API access logs?
 * Then whittle that list down to only those dependents who might be adversely affected. How many of those users have been active recently? How many have data assets in the system they may want to keep?

**Comment by gbinal** [comment]
> @gbinal Anything folks need to do on the api.data.gov side? 

Sorry for the slow response over the holidays, but thanks for checking, @afeld.  Like Tim though, I'm good.  If someone is decommissioning a system, they don't need to do anything in particular for us.  We have other workflows that'll catch anything we need to do on our end.

**Comment by afeld** [comment]
> There's a wider topic here about no longer being a dependency and removing the project in as thoughtful and non-damaging a way as possible

@yozlet Yeah, I agree...and therefore don't want the checklist to get bogged down in it. Adjusted the wording to indicate "you should put some thought into this task". Good enough?

**Comment by afeld** [comment]
Good to merge? Important to remember that we can iterate on this - doesn't have to be perfect 😉

**Comment by its-a-lisa-at-work** [comment]
@afeld I think we should make sure we capture what @yozlet said for the work we're doing for DI&E since this has ties to that.

**Pull request merged**